cyvcf2/cyvcf2.pyx

```diff
@@ -929,7 +930,7 @@ cdef class VCF(HTSFile):
         for sj in range(ns):
             sample_j = samples[sj]
             if _hets[sj] == 0:
-                print("peddy: no hets found for sample %s\n" % sample_j, file=sys.stderr)
+                warnings.warn("peddy: no hets found for sample %s" % sample_j)
```

Comment:
Nor this?

Reply:
same. leave all the peddy stuff in (but changing to warning is great).